src/garage/torch/_functions.py

```diff
@@ -187,6 +188,23 @@ def flatten_batch(tensor):
     return tensor.reshape((-1, ) + tensor.shape[2:])
 
 
+# pylint: disable=missing-return-type-doc
```

Comment:
i think you are getting this message because you didn't specify a return type.

isn't it just

```python
"""
...

Returns:
    torch.Tensor: Reshaped view of that data (analogous to numpy.reshape)
"""
```